src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -1198,26 +1198,13 @@ void*
 cSTIR_objectiveFunctionGradient(void* ptr_f, void* ptr_i, int subset)
 {
 	try {
-		ObjectiveFunction3DF& fun = objectFromHandle< ObjectiveFunction3DF>(ptr_f);
+		xSTIR_ObjFun3DF& fun = objectFromHandle<xSTIR_ObjFun3DF>(ptr_f);
 		STIRImageData& id = objectFromHandle<STIRImageData>(ptr_i);
```

Comment:
could use the shorter
```suggestion
		auto& fun = objectFromHandle<xSTIR_ObjFun3DF>(ptr_f);
		auto& id = objectFromHandle<STIRImageData>(ptr_i);
```

Reply:
done